examples/simple/lint/L-STDLIBE2-good.c

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main()
+{
+    size_t array_size = 20;
```

Comment:
for now, I've kept size_t in both good and bad versions. We can change it if needed.